datafusion/core/benches/sql_planner.rs

```diff
@@ -249,7 +249,7 @@ fn criterion_benchmark(c: &mut Criterion) {
     c.bench_function("physical_plan_tpch_all", |b| {
```

Comment:
I think it would be useful to create another function that creates the logical plan as well

```rust
c.bench_function("logical_plan_tpch_all", |b| {
        b.iter(|| {
            for sql in &all_tpch_sql_queries {
                logical_plan(&ctx, sql)
            }
        })
    });
```

I am happy to do this as a follow on PR as well.

Reply:
Added this test, thanks @alamb